content/dogepedia/articles/how-does-dogecoin-work.md

```diff
@@ -5,4 +5,12 @@ date = "2021-10-22"
   name = "Dogecoin"
 +++
  
-A high-level overview of how Dogecoin works. ELI5 style.
\ No newline at end of file
+Magic and memes! (Just kidding. Kind of.) 
+
+First things first: Dogecoin *is* money.  
+
+Like other popular cryptocurrencies, Dogecoin works by utilising [blockchain technology](https://dogecoin.com/dogepedia/articles/what-is-a-blockchain). Unlike a lot of other popular cryptocurrencies, though, the fees to send Dogecoin are extremely small, and the speed by which Dogecoin transactions are confirmed is relatively fast. 
+
+Simply put, if you would like to send money to someone who is located anywhere in the world — or receive money from someone who is located anywhere in the world — *without* using a centralised bank and paying foreign currency exchange and transfer fees, then Dogecoin can work for you! 
+
+Furthermore, Dogecoin works by using it to purchase goods and services, and even for tipping people on popular social-media platforms like Reddit. See Community and Ecosystem [link to ‘Community and Ecosystem’ subsection] for more information on how Dogecoin can be used and how it works!
```

Comment:
```suggestion
Furthermore, Dogecoin works by using it to purchase goods and services, and even for tipping people on popular social-media platforms like Reddit. See [Community and Ecosystem](/dogepedia/#community-and-ecosystem) for more information on how Dogecoin can be used and how it works!
```

Reply:
This is how to link. I am sending a separate PR to update the index file to include other in-page links.

Reply:
Ah - gotcha! Thanks!